spec/std/isa/inst/Zihintntl/c.ntl.p1.yaml

```
@@ -0,0 +1,26 @@
+# Copyright (c) Jordan Carlin
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+# yaml-language-server: $schema=../../../../schemas/inst_schema.json
+
+$schema: inst_schema.json#
+kind: instruction
+name: c.ntl.p1
+long_name: Compressed non-temporal locality hint, innermost private
+description: |
+  The NTL.P1 instruction indicates that the target instruction does not exhibit
```

Since there's no context here as to what "target instruction" refers, let's be more explicit, from the spec:

```suggestion
  The NTL.P1 instruction indicates that the immediately subsequent instruction does not exhibit
```

This comment applies to all of the hints.

---

Done